api/v1beta1/openstackdataplanenodeset_webhook.go

```diff
@@ -61,6 +61,13 @@ func (spec *OpenStackDataPlaneNodeSetSpec) Default() {
 		spec.Nodes[nodeName] = *node.DeepCopy()
 	}
 
+	if spec.NodeTemplate.Ansible.AnsiblePort == 0 {
+		spec.NodeTemplate.Ansible.AnsiblePort = 22
```

Comment:
why do we need to set the default port to 22? That's the ansible default already. We should just leave it unset.